domain/consensus/processes/consensusstatemanager/update_virtual.go

```diff
@@ -42,6 +42,11 @@ func (csm *consensusStateManager) updateVirtual(newBlockHash *externalapi.Domain
 		return nil, nil, err
 	}
 
+	_, err = csm.difficultyManager.UpdateDAADataAndReturnDifficultyBits(model.VirtualBlockHash)
+	if err != nil {
+		return nil, nil, err
+	}
+
```

Comment:
Is this needed for the `CalculatePastUTXOAndAcceptanceData`?

Reply:
Yes